src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -39,7 +39,7 @@ data class DraftReferralDTO(
         id = referral.id,
         createdAt = referral.createdAt,
         completionDeadline = referral.completionDeadline,
-        complexityLevels = referral.complexityLevelIds?.map { ReferralComplexityLevel(it.key, it.value) },
+        complexityLevels = referral.complexityLevelIds?.map { ReferralComplexityLevel(it.key, it.value) }?.sortedBy { it.serviceCategoryId },
```

Comment:
This ordering makes me nervous -- does it matter?

Reply:
I will reorder based on selectedServiceCategory.name